corehq/apps/receiverwrapper/tests/test_auth.py

```diff
@@ -111,6 +128,40 @@ def test_digest(self):
         )
 
     @run_with_all_backends
```

Comment:
Does this need to be run on both backends? I don't think there is any difference the the code that gets executed.